macros/spec2.ejs

```diff
@@ -177,6 +177,7 @@ var status = {
   'Media Capture DOM Elements' : 'ED',
   'Media Session'              : 'ED',
   'Media Source Extensions'    : 'CR',
+  'MediaStream Image'          : 'ED',
```

Comment:
I believe this should be 'WD' as there is https://www.w3.org/TR/image-capture/

See also this comment https://github.com/mozilla/kumascript/pull/245#issuecomment-316635641

> I thought spec2 was supposed to reference the furthest position along the standards track the spec had reached, which is not necessarily the same as the status of the spec referenced in specname?